brambling/utils/model_tables.py

```diff
@@ -713,3 +727,9 @@ def housing_categories(self, obj):
 
     def balance(self, obj):
         return format_money(obj.balance or 0, self.event.currency)
+
+    def completed_date(self, obj):
+        if obj.completed_date:
+            return obj.completed_date.strftime("%Y-%m-%d %H:%M")
```

Comment:
Right, if we're fine with not having it in the template, then I can do the same thing as with the finances table.